lib/hw_unique_key/hw_unique_key_kmu.c

```diff
@@ -11,22 +11,36 @@
 
 #define KMU_KEYSLOT_SIZE_WORDS 4
 
+// Use 'copy_of_uicr_word_read' instead of 'nrfx_nvmc_uicr_word_read'
+// until it has become available everywhere. Everywhere consists of
+// TF-M's copy of nrfx, nrfx, hal_nordic in Zephyr and NCS.
```

Comment:
What is meant by "Everywhere consists of TF-M's copy of nrfx, nrfx, hal_nordic in Zephyr and NCS."?

Reply:
The function is needed by TF-M upstream's local copy of nrfx.

The upstream nrfx.

And the nrfx that is copied into <west_dir>/hal_nordic for Zephyr and NCS.